### ISSUE TITLE
Minion ipv6 option is not documented

### ISSUE THREAD

**Issue by bobrik** [saltstack/salt#39118]
### Description of Issue/Question

Salt master docs: https://docs.saltstack.com/en/latest/ref/configuration/master.html

![image](https://cloud.githubusercontent.com/assets/89186/22519970/5f4ad232-e867-11e6-99ae-c1ddb5b8602c.png)

Salt minion docs:

![image](https://cloud.githubusercontent.com/assets/89186/22519993/70fc4768-e867-11e6-8161-ac9f08df48d7.png)

Salt minion docs say that `master` is `ipv4` address and don't mention `ipv6`.

When running `salt-call --master=ipv6.google.com test.ping` salt fails:

```
[ERROR   ] DNS lookup of 'False' failed.
[ERROR   ] Master hostname: 'ipv6.google.com' not found. Retrying in 30 seconds
```

This is because even though `salt-call` accepts `--master` it uses `ipv6: false` (the default).

### Versions Report

```
Salt Version:
           Salt: 2016.3.4

Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 2.2
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: Not Installed
      pycparser: 2.10
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.9 (default, Jun 29 2016, 13:08:31)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.4.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5

System Versions:
           dist: debian 8.6
        machine: x86_64
        release: 4.4.44
         system: Linux
        version: debian 8.6
```

**Comment by bobrik** [comment]
I can trade the docs PR for a fix for #38758 :)

**Comment by gtmanfred** [comment]
We are commited to fixing that in the 2016.11.3 release, Mike is working on it :D

**Comment by rallytime** [comment]
Fixed with #39131.

**Comment by bobrik** [comment]
@gtmanfred, I did my part, but #38758 doesn't seem to be marked for 2016.11.3.

**Comment by bobrik** [comment]
@gtmanfred I'm afraid 2016.11.3 is going out without a fix for #38758 ☹️

**Comment by gtmanfred** [comment]
It looks like it was delayed to 2016.11.4, but it will be in there.

Thanks,
Daniel
